src/main/java/com/rest/hw/app/service/FileService.java

```diff
@@ -0,0 +1,57 @@
+package com.rest.hw.app.service;
+
+import lombok.AllArgsConstructor;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.*;
+
+@Service
+@AllArgsConstructor
+public class FileService {
+    PersonService personService ;
+    public void writer() {
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter("dataFile", true));
+            writer.write("\n" + personService.getAll() + "\n");
+            writer.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void uploadFile(MultipartFile file) throws IOException {
+        writer();
+        File file1 = new File("//src//main//java//com//rest//app//");
+            file.transferTo(file1);
+    }
+
+    public void deleteFile(MultipartFile file) throws IOException {
+        File file1 = new File("//src//main//java//com//rest//app//");
```

Comment:
If something is used widely within your application, extract it to a constant variable